mypyc/primitives/int_ops.py

```diff
@@ -150,8 +150,15 @@ def int_unary_op(name: str, c_function_name: str) -> CFunctionDescription:
     c_function_name='CPyTagged_IsEq_',
     error_kind=ERR_NEVER)
 
+int_less_than_ = c_custom_op(
+    arg_types=[int_rprimitive, int_rprimitive],
+    return_type=bool_rprimitive,
+    c_function_name='CPyTagged_IsLt',
```

Comment:
Should this have a `_` suffix?

Reply:
It should. It's like the `int_equal_` several lines above it, which provides a CallC description for a logical op but not complete and should always be used within ll_builder's function.

Reply:
yeah, it definitely should! Thanks for catching this potentially costly bug!